docs/hiring-guide/index.rst

```diff
@@ -100,3 +100,22 @@ Questions for curious candidates to ask of interviewers
 * Where do you see the main focus of this role sitting, seen on a triangle with Users, Business, and Technology as the points?
 * What tech/business resources would you allocate to support the documentation?
 * Would you prefer an employee with deep knowledge or broad knowledge?
+
+Useful skills for a technical writer
+------------------------------------
+Develop your capabilities with these professional skills and tools.
+
+Skills and Knowledge:
+
+* Be bold. Own what you do. Not specifically a techwriting skill, but in this field we often need to prove the importance of our work.
+* Be a team worker. You definitely need to communicate with others to get the information required to perform your job. Experts are usually strong personalities; you’ll need to find a path to their heart and mind.
```

Comment:
Word choice:

"Experts often have strong personalities; ..."

Reply:
Hadas reviewed this setence and rewrote the phrase like in my text. I would keep it as is.